src/graphql/newbie-select.graphql.ts

```diff
@@ -11,7 +11,6 @@ export const NEWBIE_SELECT = gql`
         id
         photo
         name
-        startDate
```

Comment:
Already asked on top about this matter

Reply:
Answered [here](https://github.com/BuildIt-Poland/buddy-app/pull/50#discussion_r358808806)